RetailCoder.VBE/Inspections/ImplicitPublicMemberInspectionResult.cs

```diff
@@ -45,14 +45,13 @@ public override void Fix()
             var oldContent = Context.GetText();
```

Comment:
Is it just me or is this fixable by getting comments into the ParseTree as nodes?  
That would allow skipping irrelevant content or rather readding it correctly